### PR TITLE
SCIM 1.0.2 updates

### PR DESCRIPTION
**Pull request by Karuna-Mendix**

### CODE REVIEW

File: content/en/docs/appstore/platform-supported-content/modules/scim.md [hunk 1]
```diff
@@ -24,56 +24,55 @@ Before you include the SCIM module in your app, you need to check if your IdP su
...
 
 ### Features {#features}
 
-Your IdP can perform create, read, update and delete (CRUD) operations on the users in your app when it includes the SCIM module. It has the following features:
+Your IdP can perform create, read, update, and delete (CRUD) operations on the users in your app when it includes the SCIM module. It has the following features:
 
 * Create users: users who are in Entra ID and assigned to the Mendix SCIM application in Azure are automatically created in the Mendix application.
 
-* Remove users: deactivates users in the Mendix app when they are deleted from Entra ID or removed from the group of users assigned to use your app.
+* Remove users: user deletion can be either a 'hard' delete, which removes user records from the app's database, or a 'soft' delete, which deactivates the Mendix user but keeps their records. If you set the flag **Default_DeleteUserPermanently** (in the **Acceptance Environment Details** of the Mendix application environment) to *True*, the user will be hard deleted. By default, the flag is set to *False*.
```
Comment (JaapF) [inline content/en/docs/appstore/platform-supported-content/modules/scim.md:39]: First sentence is the 'what', second sentence is the 'how'.
I suggest we move the how further down in the document to keep the list of features short.

File: content/en/docs/appstore/platform-supported-content/modules/scim.md [hunk 2]
```diff
@@ -24,56 +24,55 @@ Before you include the SCIM module in your app, you need to check if your IdP su
...
 
 * The following user attributes are supported during the creation or updating of users: first name, last name, and email address.
 
 * Update users: synchronizes changes in the user's profile in your IdP with your Mendix app, such as a change in the user’s information.
 
-* Disable (Deactivate) / Enable (activate) users: deactivates or activates users in your Mendix app when you disable or enable them in the Entra ID.
+* Disable (Deactivate) / Enable (activate) users: deactivates or activate users in your Mendix app when you disable or enable them in the Entra ID.
```
Comment (JaapF) [inline content/en/docs/appstore/platform-supported-content/modules/scim.md]: either 'deactivates or activates' or 'deactivate or activate'. grmmar: subject is the same.

File: content/en/docs/appstore/platform-supported-content/modules/scim.md [hunk 3]
```diff
@@ -24,56 +24,55 @@ Before you include the SCIM module in your app, you need to check if your IdP su
...
+* **B2E vs. B2C apps:** Many organizations use a common Identity Provider, such as Entra ID in Microsoft’s Office 365 suite, to centralize employee accounts. If you are building apps for employees (Business-to-Employee or B2E), connecting them with Entra ID is a common and easy choice. As an app developer, you can use OIDC SSO or SAML for Single Sign-On and the SCIM module for managing users. For apps designed for customers (Business-to-Customer or B2C), the use of the SCIM module may not be as obvious. If your organization uses a third-party solution, such as Entra ID to handle customer accounts, you can also use the SCIM module for that.
 
 ### Features {#features}
```
Comment (JaapF) [inline content/en/docs/appstore/platform-supported-content/modules/scim.md:33]: We need to add the other new feature as well:
Draft release notes say:
 "Support of default Deploy time Configuration via constants, avoiding the need for runtime configuration by a local admin user."

File: content/en/docs/appstore/platform-supported-content/modules/scim.md (hunk 1 above)
Comment (JaapF) [reply content/en/docs/appstore/platform-supported-content/modules/scim.md:39]: We need to ensure that all constants from Bala's document are covered in the documentation.
I checked a few and I think they're not all covered yet. Let's add thoses.
https://mendix.atlassian.net/wiki/spaces/idx/pages/3217686532/SCIM+Enhancements+Delete+User+Permanently+Language+Translations+Support+and+Deploy+time+Configuration+Support

It's my understanding that these constants are relevant both at design time (section 2.1) and can be overridden at deploytime (section 2.2). I think we've the same pattern in the draft SAML documentation for forthcoming release.

Comment (Karuna-Mendix) [reply content/en/docs/appstore/platform-supported-content/modules/scim.md:39]: I will cover the Deploy-time configuration and constants details in another PR. I will keep you posted about it, tomorrow.

File: content/en/docs/appstore/platform-supported-content/modules/scim.md [hunk 4]
```diff
@@ -47,33 +47,33 @@ Your IdP can perform create, read, update and delete (CRUD) operations on the us
...
     {{% alert color="info" %}}
 If you are using the SCIM module in combination with Entra ID and OIDC SSO, you need to choose the correct attribute mapping. For more information, see the [Guidance on User Identifier](/appstore/modules/oidc/#guidance-user-identifier) section of the *OIDC SSO* and [Attribute Mapping](#attribute-mapping) section below.
     {{% /alert %}}
 
 * Your IdP allows the selection of users that sync with your Mendix app, meaning your IdP controls which users are created and active in your app.
-* Each application within your organization using the SCIM module must undergo separate configuration. This allows the selection of the right target group of users for each app.
-* Users can be synchronized from multiple SCIM clients.
+* Each application within your organization using the SCIM module must undergo a separate configuration. This allows the selection of the right target group of users for each app.
+* Synchronization of users from multiple SCIM clients is currently in beta.
+* The module supports deployment-time configuration using constants, eliminating a need for runtime configuration by a local admin user.
```
Comment (Karuna-Mendix) [inline content/en/docs/appstore/platform-supported-content/modules/scim.md:59]: Added a feature as per the release note.

File: content/en/docs/appstore/platform-supported-content/modules/scim.md [hunk 5]
```diff
@@ -284,9 +284,11 @@ Once you have your SCIM module configured, you can test it by creating, updating
 The test case below is defined for the scope of **Sync only assigned user and groups** and validation of provisioning status in the SCIM server.
 
 * Create a user with default `userPrincipalName`. Optionally, you may choose to change the user Mappings in the Microsoft Entra ID. You can change Entra ID attribute from `userPrincipalName` to another attribute. For example, `mailNickName`.
-* Update a user in the Microsoft Entra ID. For Example, add/update work email, last name etc.
+* Update a user in the Microsoft Entra ID. For Example, add/update work email, last name, etc.
 * Delete a user in the Microsoft Entra ID.
 
+To hard delete a user, you can set the flag **Default_DeleteUserPermanently** (in the **Acceptance Environment Details** of the Mendix application environment) to *True*. By default, this flag is set to *False*.
```
Comment (Karuna-Mendix) [inline content/en/docs/appstore/platform-supported-content/modules/scim.md:290]: I hope this is the correct place to add **Default_DeleteUserPermanently** flag info. @JaapF Please suggest.

**Pull request merged**